src/swapnaWJun20/ConvertCaseASCII.java

```diff
@@ -0,0 +1,34 @@
+/*WAP to convert case in all elements of array. 
+Input : 
+String[] array = {"AaVruti", "NiKhil", "AdiTi", "MAhesh", "ArchaNA", "jaGAdeesh"};
+
+Output:
+array = {"aAvRUTI", "nIkHIL","aDItI", "maHESH","aRCHAna","JAgaDEESH"};*/
+
+package swapnaWJun20;
+
+public class ConvertCaseASCII {
+
+	void caseConvertString(String input[]) {
+		System.out.print("{");
+		for (int outerIndex = 0; outerIndex < input.length; outerIndex++) {
+			String convertString = "";
+			for (int index = 0; index < input[outerIndex].length(); index++) {
+				if (input[outerIndex].charAt(index) >= 'A' && input[outerIndex].charAt(index) <= 'Z')
+					convertString += (char) (input[outerIndex].charAt(index) + 32);
+				else
+					convertString += (char) (input[outerIndex].charAt(index) - 32);
+			}
+			input[outerIndex] = convertString;
+			System.out.print("\"" + input[outerIndex] + "\",");
+		}
+		System.out.println("}");
+	}
```

Comment:
The logic looks good. Instead of hard coding the output, you can use Arrays.toSTring. First declare an output empty array String[] array = new String[input.length] and you can add the converted strings to the index of the array. When you are having an array, to print it you can write 
Syste.out.println(Arrays.toString(array);
Hope this helps.